File: app/services/omni/receipt.rb
class Omni::Receipt::Service
  include Buildit::Service::Base

  service 'Receipt', 'Omni.service'

  connected_mode(Buildit::Service::Backend::Crud) do |config|
    config.model = Omni::Receipt
  end

end # class Omni::Receipt::Service

File: app/assets/javascripts/omni/store/Receipt.js
Ext.define('Omni.store.Receipt', {
  extend       : 'Ext.data.Store',
  alias        : 'store.omni-Receipt',
  model        : 'Omni.model.Receipt',
  autoLoad     : false,
  storeId      : 'ReceiptStore',
  remoteFilter : true,
  remoteSort   : true,

  constructor  : function (config) {
    var me = this;
    me.callParent(this);
    var proxy = Ext.Object.merge({}, me.getProxy());
    me.setProxy(proxy);
  } // constructor

}); // Ext.define